frontend/public/locales/en/projectformTranslation.json

```diff
@@ -17,5 +19,9 @@
     "clearSelected": "Clear Selection",
     "tooltipRunner": "If you're having trouble figuring out the runner please refer to the docs",
     "userDocs": "runner user docs"
+  },
+  "dragAndDrop": {
+    "dragDropHere" :"Drag & Drop a File Here",
```

Comment:
In the English version File and Here are capitalized while it's not in the Dutch version, better to keep it consistent.